code/modules/mob/hear_say.dm

```diff
@@ -9,6 +9,8 @@
 			//Or someone snoring.  So we make it where they won't hear it.
 		return
 
+	message = capitalize(message)
```

Comment:
Alright, this is clearly not the way to go about this. I'll shut this ill-conceived PR down, then.